packages/hooks/src/createHasHook.js

```diff
@@ -5,17 +5,25 @@
  * @param  {Object}   hooks Stored hooks, keyed by hook name.
  *
  * @return {Function}       Function that returns whether any handlers are
- *                          attached to a particular hook.
+ *                          attached to a particular hook and optional namespace.
  */
 function createHasHook( hooks ) {
 	/**
-	 * Returns how many handlers are attached for the given hook.
+	 * Returns whether any handlers are attached for the given hookName and optional namespace.
 	 *
-	 * @param  {string}  hookName The name of the hook to check for.
+	 * @param  {string}  hookName  The name of the hook to check for.
+	 * @param  {string}  namespace Optional. The unique namespace identifying the callback in the form `vendor/plugin/function`.
```

Comment:
Single spaces between `@param` and `{string}` and `name`

```js
	 * @param {string} hookName  The name of the hook to check for.
	 * @param {string} namespace Optional. The unique na
```

Reply:
thanks, corrected.